helm/charts/hydra/templates/janitor-cron-job.yaml

```diff
@@ -0,0 +1,103 @@
+{{- if .Values.janitor.enabled -}}
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: {{ include "hydra.fullname" . }}-janitor
+  {{- if .Release.Namespace }}
+  namespace: {{ .Release.Namespace }}
+  {{- end }}
+  labels:
+    {{- include "hydra.janitor.labels" . | nindent 4 }}
+    {{- with .Values.janitor.cronjob.labels }}
```

Comment:
Just as a general note, we tend to separate application logic from deployment logic, that is why `deployment` and `jobs` have dedicated top-level field. I would prefer to have 

```
---
# values.yaml
janitor:
  applicationLogicFields

cronjob:
  janitor:
    k8sCronSettings
```